lib/3_add_up_numbers.php

```diff
@@ -7,9 +7,34 @@
 // Run `bin/check` from the command line to execute the automated tests.
 
 function addUpNumbers($arr) {
-  // Your code goes here.
+  $total = 0;
+  for ($i = 0; $i < count($arr); $i++) {
+    $total += $arr[$i];
+  }
+  return $total;
 }
 
 // Write your own "tests" below. Refer to the the examples from exercise #1.
 
+echo "\nTesting that the array [1, 2, 3] adds up to 6.\n";
+if (addUpNumbers(array(1, 2, 3)) == 6) {
+  echo "Success!";
+}
+else{
+  echo "Failed!";
+}
+echo "\n-------------------------------------------------------------------\n";
+
+echo "\nTesting that the array [3, 2] adds up to 5.\n";
+if (addUpNumbers(array(3, 2)) == 5)  {
```

Comment:
Honestly I didn't see a whole lot of difference in the numbers being in decreasing rather than increasing order but I also didn't see much else that I could possibly put tests in for that'd be meaningful in this context.  The problem stated:

> The inputted $arr will always be an array/list like [5, 10, 2]. 

It seems misleading for the problem to say that the input will always be like an array of positive integers, and then say we should be testing for things which are not positive integers?